pavelib/quality.py

```diff
@@ -470,6 +516,17 @@ def _pep8_output(count, violations_list, is_html=False):
     ):
         diff_quality_percentage_pass = False
 
+    # run diff-quality for safelint.
+    if not run_diff_quality(
+            violations_type="safelint",
```

Comment:
@jzoldak: It seems that diff-quality actually needs to know about the tool.
```
edxapp@precise64:~/edx-platform$ diff-quality -h
usage: diff-quality [-h] --violations TOOL [--html-report FILENAME]
...
  --violations TOOL     Which code quality tool to use
                        (eslint/flake8/jshint/pep8/pyflakes/pylint)
```
Any thoughts on the best way to proceed for this?  Does this mean we need our own for of diff-cover that knows about this new safe template linter?  I had originally thought that I just needed to supply reports to this tool.  Am I missing something?  Thanks for any input on this.